README.md

```diff
@@ -83,13 +84,86 @@ Pitfalls while Systemupgrades
 
   If you don't change that value, apache2 package becomes deleted while upgrading Ubuntu.
 
+Pitfalls after Praktomat-Update (Summer 2022)
+=================
+
+A note about `prlimit`
+----------
+  To make Praktomat compatible with newer Python versions, we introduce a dependency to binary executable program `prlimit`.
+
+    util-linux (containing prlimit)
+
+  If you have installed `util-linux` but the command `prlimit` remain unknown, your linux system may be too old.
+  On some circumstances you can build `prlimit` from scources ( more about this cf. https://github.com/KITPraktomatTeam/Praktomat/pull/342#issuecomment-1219445202 )
+
+
+ A note about SQLite3 and Praktomat's unit tests
+----------
+
+Unit tests which are testing Praktomat code are using `SQLite3` as database backend, depending on settings in `Praktomat/src/settings/test.py`.
+
+Since merge of feature boxplot-taskstatistic (cf https://github.com/KITPraktomatTeam/Praktomat/pull/345 ) at 2022-07-27
+your Python environment is using an to old `SQLite3` version, if tests fails with message:
+
+
+    django.db.utils.OperationalError: near "(": syntax error while migrating migrationfile taskstatistics.0001_initial_TaskStatisticsDBview
+
+
+In example `SQLite3` version 3.16.2 dated to 2017-01-06 is much to old to handle the hand written migration file `Praktomat/src/taskstatistics/migrations/0001_initial_TaskStatisticsDBview.py` correctly.
+
+Since that migration file use *SQL window functions*, which where standarized in `SQL:2003` and covered by `SQLite3` with version 3.25.0 dated to 2018-09-15. But, because of some bugs inside `SQLite3`, you cannot use that first version supporting *SQL window functions*, too.
```

Comment:
```suggestion
In example, `SQLite3` version 3.16.2 dated to 2017-01-06 is much too old to handle the handwritten migration file `Praktomat/src/taskstatistics/migrations/0001_initial_TaskStatisticsDBview.py` correctly.

That migration file uses *SQL window functions*, which were standardized in `SQL:2003` and covered by `SQLite3` with version 3.25.0 dated to 2018-09-15. But, because of some bugs inside `SQLite3`, you cannot use that first version supporting *SQL window functions*, too.
```

When we're about to fix orthography errors... ;)